x/delayedack/genesis.go

```diff
@@ -9,17 +9,12 @@ import (
 // InitGenesis initializes the module's state from a provided genesis state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
-	for _, packet := range genState.RollappPackets {
-		err := k.SetRollappPacket(ctx, packet)
-		if err != nil {
-			panic(err)
-		}
-	}
 }
 
 // ExportGenesis returns the module's exported genesis
 func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	return &types.GenesisState{
+		Params:         k.GetParams(ctx),
 		RollappPackets: k.GetAllRollappPackets(ctx),
```

Comment:
shouldn't we export the same data used in the import, and remove unused variables from genesistate?

Reply:
We didn't want to break the `Export` functionality, as it might be useful for other things,
regardless of the ability to init new chain from it.